python/ray/_private/ray_constants.py

```diff
@@ -23,6 +23,21 @@ def env_integer(key, default):
     return default
 
 
+def env_float(key, default):
```

Comment:
QQ: Any use cases where we set float in env_var?

Reply:
Good question! In some cases, it's better to use float numbers for `timeout_values`, especially when comparing them to real-time (float) values. Reading them as floats makes more sense in those scenarios.